src/components/SavedNews.js

```diff
@@ -70,13 +70,7 @@ const SavedNews = ({
   }, [newsCards]);
 
   React.useEffect(() => {
-    getArticles(token)
-      .then((data) => {
-        setNewsCards([...new Map(data.map((v) => [v.title, v])).values()]);
-      })
-      .catch((err) => {
-        console.log(err);
-      });
+    setNewsCards([...new Map(savedCards.map((v) => [v.title, v])).values()]);
```

Comment:
Grab card from `SavedCardsContext` so no longer have to make an additional API call to fetch the card.